### DISCUSSION TITLE
Is there new release coming soon.

### DISCUSSION THREAD

**Discussion by ahmedsamer** [hassio-addons/addon-nginx-proxy-manager#488] [Q&A] [answered]
Hello,

I've noticed a huge amount of changes already on the main branch for this plugin, While the latest release v0.12.3 was on  Nov 17, 2022.

So my question will be, Is there a new release coming up in the near future? or is there any way to use the edge version inside Homeassistant?

Regards

**Comment by frenck** [comment, marked as answer]
> So my question will be, Is there a new release coming up in the near future? 

Yes, I'm currently awaiting a change in the Supervisor to be released stable, which provides some new functionality I'm going to leverage.

> or is there any way to use the edge version inside Homeassistant?

There is, but it is broken at this point awaiting the above-mentioned change.

This new functionality I need, is expected to be generally available in stable this week or beginning of next week.

../Frenck

**Reply by ahmedsamer** [reply]
Thanks a lot, Frenck.

**Comment by frenck** [comment]
I'm (hopefully) happy enough with testing to ship a new version this weekend.

../Frenck
